wolfengine/README.md

```diff
@@ -0,0 +1,42 @@
+# wolfSSL Open Source Project Ports with wolfEngine
+
+The goal of the wolfEngine project is to replace the crypto used by OpenSSL
+with wolfCrypt or wolfCrypt FIPS, with the least amount of changes possible.
+
+Each of the project directories has both a patch file and a reference to a
+submodule for the project at the commit for the patch. After cloning the OSP
+repository, change directories into the project's subdirectory. Run the
+command `git submodule update && git submodule update`. Or if the OSP
+repository hasn't been cloned yet, clone it with the command:
+
+    git clone --recurse-submodules https://github.com/wolfSSL/wolfEngine.git
+
+Change directories into the project's directory and run
+`patch -p1 ../PROJ-patch.txt` to apply the patch file. At this point, build
```

Comment:
Missing `<` in patch command .  `patch -p1 < ../PROJ-patch.txt`

Reply:
Got it, thanks.